src/main/java/seedu/address/model/Model.java

```diff
@@ -90,6 +96,14 @@
      */
     void addGroup(Group group);
 
+    /**
+     * Adds the task into the specified group.
+     * {@code group} must already exist in the address book.
+     * {@code task} must not already exist in the address book.
```

Comment:
As mentioned earlier about description of unique task. But correct me if I'm wrong 😄